components/automate-load-balancer/habitat/config/nginx.conf

```diff
@@ -111,8 +111,10 @@ http {
     ssl_certificate {{../pkg.svc_data_path}}/{{tls.server_name}}.cert;
     ssl_certificate_key {{../pkg.svc_data_path}}/{{tls.server_name}}.key;
 
-    ssl_client_certificate {{../pkg.svc_config_path}}/root_ca.crt;
-    ssl_verify_client optional;
+    {{#if ../cfg.ngx.http.ssl_client_certificate_filename ~}}
+    ssl_client_certificate {{../pkg.svc_config_path}}/{{ ../cfg.ngx.http.ssl_client_certificate_filename }};
```

Comment:
have you tested this. I'm fairly certain there is no way to make this work with authz